### ISSUE TITLE
Reset history?

### ISSUE THREAD

**Issue by bonesoul** [CartBlanche/MonoGame-Samples#28]
The repo is so large now (around 479 mb of download), should we consider resetting the history?

**Comment by CartBlanche** [comment]
@bonesoul it is certainly something worth considering.

**Comment by NicusorN5** [comment]
580 MBs now. Quite the fun downloading this

**Comment by SimonDarksideJ** [comment]
I would say so. the change history isn't providing any value at this point until the project is fully functional.
Wipe it :D

**Comment by NicusorN5** [comment]
Yes. I noticed that the shaders are incompatible with Monogame...

**Comment by CartBlanche** [comment]
Once everything is bumped to being 3.8.4 nugets and SDK projects, I'll reset the history.

**Comment by CartBlanche** [comment]
New `main` branch has had it's history reset and now has all projects updated to use SDK projects and at the very least DesktopGL/Windows projects build and run with MonoGame 3.8.4 and above. 

Even though there are Android and iOS projects there, the majority may now work. A future PR will address getting Android and iOS builds working. For the most people. Closing this issue.
